.github/workflows/CI-integrationtests.yml

```diff
@@ -22,6 +22,9 @@ jobs:
     container:
       image: ghcr.io/gammasim/simtools-dev:latest
       options: --user 0
+    strategy:
+      matrix:
+        type: ['2024-02-01', 'prod5']
```

Comment:
I was expecting this to run over prod6 as well. Is there a reason why it is not?

Reply:
Oh, 2024-02-01 is a prod6 prototype implementation. The model needs some more testing and some update, so better not to call it prod6.

Reply:
ok

Reply:
I tested now running the test for model_version="prod6" `pytest --no-cov --model_version="prod6" "tests/integration_tests/test_applications_from_config.py::test_applications_from_config[simtools-print-array-elements_utm_to_ground_meta_in_yml]"` and it works (the test also passes). I guess this is because we already have the model parameters in the DB?

Reply:
Exactly. There is no difference between 2024-02-01 and prod6 (this is resolved in `names.py`)